modules/seqexec/server/src/main/scala/seqexec/server/tcs/TcsController.scala

```diff
@@ -142,6 +163,20 @@ object TcsController {
         NodChopTrackingOption.fromBoolean(s.exists(_ === nodchop))
     }
 
+    implicit val specialEq: Eq[Special] = Eq.by(_.s)
+
+    implicit val activeEq: Eq[ActiveNodChopTracking] = Eq.instance{
+      case (Normal, Normal)             => true
+      case (a@Special(_), b@Special(_)) => a === b
+      case _                            => false
+    }
+
+    implicit val eq: Eq[NodChopTrackingConfig] = Eq.instance{
+      case (None, None)                                         => true
```

Comment:
This took me a while to understand. I thought you were referring to `None` from scala Option. Perhaps we should use another name